addons/sale_timesheet/static/src/components/so_line_field/so_line_field.js

```diff
@@ -3,6 +3,10 @@
 import { registry } from "@web/core/registry";
 import { Many2OneField } from "@web/views/fields/many2one/many2one_field";
 
+// ensure load order
+// TODO Remove me when the gantt view is converted in OWL
+import "@web/views/fields/x2many/x2many_field";
```

Comment:
Yes, that was the weird issue I had yesterday about registries not being populated in the right order.

If I have to guess it's because I import PropertyValue in list_renderer but I couldn't really test that. Tests definitely don't pass without this though, and there's another pre-existing TODO next to the problematic line in the file so they should be removed together, hopefully...

Reply:
ok :) let's keep an eye on this TODO :)